website/defektoskop.php

```diff
@@ -44,7 +44,7 @@ function success($tablica, $logerrors = true, $error_msg = '', $severity = '1',
 background:white;
 padding:10px;
 margin-top:40px;
-	'><img src='".CONFIG_CDN_IMAGES."/success.png' style='display: block; margin-left: auto; margin-right: auto; padding-bottom:10px;'>";
+	'><img src='".CONFIG_CDN_IMAGES."/icons/success.png' style='display: block; margin-left: auto; margin-right: auto; padding-bottom:10px;'>";
```

Comment:
Sorry could you remind me which repo is behind https://cdn.geokrety.org ??

Do you confirm, that CDN refers to https://github.com/geokrety/GeoKrety-Static/tree/master/content/ ?

If Yes I got 1 comment:
-`success.png` seems missing, whereas others ressources updated here exists

OK

https://cdn.geokrety.org/images/icons/note10.png

KO

https://cdn.geokrety.org/images/icons/success.png

Reply:
> Do you confirm, that CDN refers to https://github.com/geokrety/GeoKrety-Static/tree/master/content/ ?

Exactly. Good catch!!!